rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] Getters and Setters for @book and @person instance variables should be present.